README.md

```diff
@@ -163,6 +163,10 @@ GitHub to deliver. For example, the default `Just the push event` works well
 with the email diff sending per-repo program above, but you can specify
 whichever events you wish.
 
+Once you have set up your webhook, GitHub will automatically send a test "ping"
+message. snare acknowledges pings (i.e. confirms to GitHub that it is running),
```

Comment:
I think "snare" is backticked through this README, so we need to do that here.

Reply:
fixed